src/web_app/src/pages/drl_hyperParams.tsx

```diff
@@ -10,13 +10,37 @@ import Grid from "@mui/material/Grid";
 import TextField from "@mui/material/TextField";
 import Stack from "@mui/material/Stack";
 import Button from "@mui/material/Button";
-import * as Types from "../types/data";
 import axios from "axios";
 import Backdrop from "@mui/material/Backdrop";
-import sleep from "../config/sleep_funct";
 import CircularProgress from "@mui/material/CircularProgress";
 import FormControlLabel from "@mui/material/FormControlLabel";
 import Checkbox from "@mui/material/Checkbox";
+const datasets = [
```

Comment:
This const value should not be here anymore

Reply:
I use it to get the corresponding dataset by taking as argument the dataset number in the url.

Reply:
This can be a problem for future addition of new datasets, in any case, this list is duplicated in many files, can you at least factor the list out to reduce the problem ?

Reply:
> You can fetch the list of datasets using the `localhost:5001/benchmarks/` endpoint instead.

This is not possible, the list is used here as the benchmark is passed from page to page via an integer representing the index of the benchmark in the list. The problem is : the server does not guarantee the order of the benchmarks when responding to this route.